test/testdrive/kafka-exactly-once-sink.td

```diff
@@ -360,3 +360,10 @@ $ kafka-verify format=json sink=materialize.public.json_avro_upsert_key key=true
 
 $ kafka-verify format=json sink=materialize.public.json_avro_upsert_key_2 key=true
 {"b": 2} {"column1": 1, "b": 2, "column3": 3, "transaction": {"id": "0"}}
+
+# Verify compaction of exactly once sinks.  Add sleep to make sure we have at least one extra binding being computed.
+$ sleep-is-probably-flaky-i-have-justified-my-need-with-a-comment duration=1
```

Comment:
Is the test there to ensure that we have at least some bindings? Without it, can it happen that we have zero bindings? If yes, you count change `verify-timestamp-compaction` to be sth like `verify-timestamp-bindings <min> <max>` to ensure that we have some, but not too many. And I think verifiers can be retried

Reply:
I'm not sure that we actually need / want to require that there are timestamp bindings?  If the system were to work correctly without any bindings (instead of the 1 or 2 we expect), that would be okay.  The issue with the max size is that it has perf impact that I don't think we have any reason to believe 0 bindings would.  What do you think?

I will look into having the verification retry since it's currently failing in CI and perhaps we just need slightly more time for the compaction to happen on a CI machine vs my local machine

Reply:
That's a good point! Then making the verification retryable should work, because we're not at danger of having to few bindings.